README.md

```diff
@@ -32,82 +32,57 @@ Full documentation is not yet hosted online but can be built and hosted locally
+section below, you first you must initialize the client with your Planet 
+username and password:
+
+```
+% planet auth init
+Email: <youremail@domain.com>
+Password: 
+Initialized
 ```
 
-Not into async? No problem. Just wrap the library and async operations together
-and call from your synchronous code.
+The email address and password you use should be the same as your login to 
+[Planet Explorer](https://planet.com/explorer). The `auth init` command
+will automatically get your API key and store it locally.
 
-```python
->>> def sync_create_order(order_details):
-...     return asyncio.run(create_order(order_details))
->>>
->>> oid = sync_create_order(order_details)
+Now that you're initialized let's start with creating an order with the 
+Orders API:
 
 ```
+% planet orders create --name my-first-order --id <scene ids> \ 
+  --item-type SkySatCollect --bundle visual
+```
+
+You should supply a unique name after `--name` for each new order, to help
+you identify what oder. The `--id` is one or more scene ids (separated by
+commas). These can be obtained from the data API, and you can also grab them
+from any search in Planet Explorer. Just be sure the scene id matches the
+[item-type](https://developers.planet.com/docs/apis/data/items-assets/#item-types) 
+to get the right type of image. And then be sure to specify a 
+[bundle](https://developers.planet.com/docs/orders/product-bundles-reference/).
+The most common ones are `visual` and `analytic`. 
 
-When using `asyncio.run` to develop synchronous code with the async library,
-keep in mind this excerpt from the
-[asyncio.run](https://docs.python.org/3/library/asyncio-task.html#asyncio.run)
-documentation:
-
-"*This function always creates a new event loop and closes it at the end. It
-should be used as a main entry point for asyncio programs, and should ideally
-only be called once.*"
-
-Do you have a use case where native synchronous support is essential? If so,
-please contribute to this
-[discussion](https://github.com/planetlabs/planet-client-python/issues/251).
-
-Why async? Because things get *really cool* when you want to work with multiple
-orders. See [orders_create_and_download_multiple_orders.py](examples/orders_create_and_download_multiple_orders.py) for
-an example of submitting two orders, waiting for them to complete, and
-downloading them. The orders each clip a set of images to a specific area of
-interest (AOI), so they cannot be combined into one order.
-(hint: [Planet Explorer](https://www.planet.com/explorer/) was used to define
-the AOIs and get the image ids.)
+This will give you an order response JSON as shown in the 'example response' in
+[the docs](https://developers.planet.com/docs/orders/ordering/#basic-ordering). You can 
+grab the `id` from that response and then use that in a single command to wait for
+the order and download it when you are ready:
+
+```
+% planet orders wait dfdf3088-73a2-478c-a8f6-1bad1c09fa09 && planet orders \
```

Comment:
What about this approach?

```bash
$ ORDER_ID=a169161d-fab5-4c89-8708-91458d7a9077 && \
    planet orders wait $ORDER_ID && \
    planet orders download $ORDER_ID
```

Reply:
Cool, will try it out and see if it works.

Reply:
Seems to be working! This is making me think we really need some sort of docs that introduce piping and && and explain how they work (or a link to a really good resource if that exists). Since I'm not yet sure why yours worked and mine didn't, and we're going to be leaning hard on the ability to do these things. I think it's fine if people don't want to learn it and we just have enough usage examples that they can do the main things, but also give people a way to learn. cc @JuLeeAtPlanet

Reply:
Hrm, it worked for a bit:

```
% ORDER_ID=c40af194-f005-45a9-822a-642f34a62062 && \
    planet orders wait $ORDER_ID && \
    planet orders download $ORDER_ID
00:25 - order c40af194-f005-45a9-822a-642f34a62062 - state: running
Error: Maximum number of attempts (5) reached.
```

Reply:
worked!